T8Suite/Form1.cs

```diff
@@ -145,6 +142,8 @@ public partial class Form1 : Form
         private string logworksstring = LogWorks.GetLogWorksPathFromRegistry();
         private static Logger logger = LogManager.GetCurrentClassLogger();
 
+        private enum Age { Old, New };
```

Comment:
Not so easy to know for you, but I think CalibrationType or something would be better than Age